### ISSUE TITLE
Allow publishing the documentation on ReadtheDocs

### ISSUE THREAD

**Issue by miceno** [collectiveaccess/ProvidenceDocs#5]
I suggest publishing the content of the help in a site like ReadtheDocs. 

I have created a sample deployment at: https://providence.readthedocs.io/en/latest/, since the Wiki documentation is down (https://docs.collectiveaccess.org/) and I found no other way to use documentation.

If the community considers it is worth publishing the docs at ReadtheDocs, I will delete the sample project to release the domain.

**Comment by miceno** [comment]
This issue is related to #3 , since in order to publish in ReadtheDocs, you need a requirements.txt file.

**Comment by collectiveaccess** [comment]
We're using the Christmas holiday to rebuild the server the documentation is on. Docs.collectiveacces.org is a MediaWiki site. This repository contains the static manual that will eventually replace the docs.collectiveaccess.org wiki.

We have no problem publishing the new manual on readthedocs, but while it is in development our main concern has been keeping it available at https://manual.collectiveaccess.org to support our work.

We'll see about adding the requirements.txt file after the holiday.

Merry Christmas!

**Comment by miceno** [comment]
Great! Thank you! I tried to read any news about the downtime of the Wiki server, but as there wasn't, I thought it might be an issue.

I am a volunteer on an historical association in Spain. I found CA software and I am building a prototype for my institution during Christmas holidays.

I am trying to access the new site, but it is not working at https://manual.collectiveaccess.org (HTTP 403). In the meantime will continue using the ReadtheDocs site.
